README.md

```diff
@@ -31,16 +31,17 @@ Environment variables
 
 The following environment variables are exposed for InfraMock:
 
-`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. If this is
-set to `true`, InfraMock will try to use a locally running version of `api` to populate the
-localstack DynamoDB database with using its `/experiments/generate` endpoint. For this to work,
-`api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior. The default value is `true`.
+`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. This is
+set to `true` by default, which means that InfraMock will try to use a locally running version of 
+the `api` to populate the localstack DynamoDB database with using its `/experiments/generate` endpoint. 
+For this to work, `api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior.
 
 `MOCK_EXPERIMENT_DATA_PATH`: where to get the mocked data for upload to the local S3 from. If
-this is not set, it will default to the GitLab URL where the data set used for the `worker` unit
+this is not set, it will default to the Github URL where the dataset used for the `worker` unit
 test is located.
 
-`AWS_DEFAULT_REGION`: the default mocked region for your infrastructure to be deployed under. If it's not set, it defaults to `eu-west-1`.
+`AWS_DEFAULT_REGION`: the default mocked region for your infrastructure to be deployed under. If it's not set, 
+it defaults to `eu-west-1`.
 
 FAQ
```

Comment:
@marcellp under the FAQ, for the first question, if one wanted to connect to `staging` or `production`, do they still need to be running inframock? Because if not, then this question and its answer should be moved elsewhere. If yes, then we need to update it with the steps required from inframock side.

Reply:
You never needed to run InfraMock for that, I don't understand why I put it in here. This is more of a general FAQ for a global documentation.

Reply:
thanks for clarifying, there is already duplicate docs in the api for this, so I will just remove it from here